README.md

```diff
@@ -59,6 +59,19 @@ $ go install github.com/google/osv-scanner/cmd/osv-scanner@v1
 
 This requires Go 1.18+ to be installed.
 
+### Building
+
+You can build from source using [GoReleaser](https://goreleaser.com/) and running:
+
+```bash
+$ goreleaser build --rm-dist --single-target --snapshot
+```
+
+See GoReleaser [documentation](https://goreleaser.com/cmd/goreleaser_build/) for build options.
+
+You can also reproduce the downloadable builds by checking out the specific tag and calling `goreleaser build`,
```

Comment:
I think "running" makes more sense here, and is consistent with the first sentence in this section:

```suggestion
You can also reproduce the downloadable builds by checking out the specific tag and running `goreleaser build`,
```

Reply:
Indeed, thanks.